tests/cypress/e2e/actions_users/issue_1810_login_logout.js

```diff
@@ -18,7 +18,6 @@ context('When clicking on the Logout button, get the user session closed.', () =
     }
 
     before(() => {
-        // TMP fix for login tests, need to change login logic with sessions
```

Comment:
Should we remove also:
```
cy.clearAllCookies();
        cy.clearAllLocalStorage();
```

???

Reply:
That paticular test depends on const task. `Prepare to testing` logins and creates task, and then user is loggined, and we cant login once again without logout, so we need to clear data before it. Frankly, im not sure why this test was passing before.